projects/css-custom-vars-viewer/src/DemoApp.ts

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I used this because it says `property` is an unused variable, but I'm using it, so I was confused why it was erroring. It wasn't allowing me to commit for this reason. This happens both here and on the standalone project's git.